rules.md

```diff
@@ -334,7 +334,7 @@ Then you can reserve the Task by leaving a comment on the relevant Trello Card o
 
 ```I want to reserve Production Tasks 1-3```
 
-An Admin will assign your Trello user in the Task on the Card along with a due date for completion, or will discuss any questions or issues with the reservation in the comments.
+An Admin (typically the Bounty's Primary Admin) will assign your Trello user in the Task on the Card along with a due date for completion, or will discuss any questions or issues with the reservation in the comments.
```

Comment:
Introduces ambiguity? Could it be clearer when/whether this is the role of the Primary or Secondary Admin?

Reply:
This is another case of not wanting to repeat logic/rules in two different sections.  Since we have a dedicated Admin section I was leaning toward putting detail there.  This does add ambiguity if someone is reading from top to bottom and only one section for a given topic.

Personally, I think the solution to this is to restructure the document to do away with a dedicated Admin section at the end, and instead include a brief definitions section at the top and put all admin-related detail into the sections they relate to.  That way there's neither overall redundancy nor sectional ambiguity.  But again, that's out of scope.  

How about this for now:
> An Admin will assign you to the Task on the Card along with a due date for completion, or will discuss any questions or issues with the reservation in the comments.  Primary Admins are responsible for assigning users to tasks except when the Primary Admin is the user requesting a task, in which case the Secondary Admin makes the assignment. 

(As a side note, I think we should do away 'document talking to reader' tenses (i.e. using "you") at some point.  I just kept it in this edit to maintain the flow established above.  In a different rule update we could consider replacing "you" references with 3rd person replacements, e.g. "the requesting Trello user" in this case.)

Reply:
Yes, I agree, this is probably a matter for a future PR addressing some of the existing grammatical & other idiosyncrasies, rather than the subsatnce of your changes. We can refer back to it when addressing those at a later date.